package-lock.json

```diff
@@ -10552,7 +10552,7 @@
       "version": "0.0.8",
       "resolved": "https://registry.npmjs.org/ansi-html-community/-/ansi-html-community-0.0.8.tgz",
       "integrity": "sha512-1APHAyr3+PCamwNw3bXCPp4HFLONZt/yIH0sZp0/469KWNTEy+qN5jQ3GVX6DMZ1UXAi34yVwtTeaG/HpBuuzw==",
-      "devOptional": true,
+      "dev": true,
```

Comment:
why is package-lock changing if `package.json` is unchanged?

Reply:
good question 😑